src/main.js

```diff
@@ -1,23 +1,28 @@
+//Book Cover HTML elements
 var coverImage = document.querySelector(".cover-image");
 var title = document.querySelector(".cover-title");
 var tagLine = document.querySelector(".tagline");
```

Comment:
It looks like this variable/querySelector is not being used in the project, so you can remove it